docs/installation/docker_compose.rst

```diff
@@ -0,0 +1,75 @@
+.. _installation_docker_compose:
+
+============================
+Install using Docker Compose
+============================
+
+This installation is meant for people who want to just try out Open Forms on 
+their own machine.
+
+A `Docker Compose`_ file is available to get the app up and running in minutes.
+It contains 'convenience' settings, which means that no additional 
+configuration is needed to run the app. Therefore, it should **not** be used 
+for anything other than testing. For example, it includes:
+
+* A default ``SECRET_KEY`` environment variable
+* A predefined database with the environment variable 
+  ``POSTGRES_HOST_AUTH_METHOD=trust``. This lets us connect to the database 
+  without using a password.
+* Debug mode is enabled.
+
+.. _`WSL`: https://docs.microsoft.com/en-us/windows/wsl/
+
+Prerequisites
+=============
+
+You will only need Docker tooling and nothing more:
+
+* `Docker Engine`_ (Desktop or Server)
+* `Docker Compose`_ (sometimes comes bundled with Docker Engine)
+
+.. _`Docker Engine`: https://docs.docker.com/engine/install/
+.. _`Docker Compose`: https://docs.docker.com/compose/install/
+
+
+Getting started
+===============
+
+1. Create a folder to store everything in:
+
+   .. code:: bash
+
+      $ mkdir open-forms
+      $ cd open-forms
+
+2. Download the ``docker-compose.yml`` file:
+
+   .. code:: bash
+
+      $ wget https://raw.githubusercontent.com/open-formulieren/open-forms/master/docker-compose.yml
```

Comment:
This is not sufficient, as some config files are mounted as container volumes that aren't downloaded if you only download the docker-compose (currently only the nginx server listening on port 9000).

I'd change this to downloading the zip or a git clone.

Reply:
I really wanted to avoid that but with wgetting the 3 docker-* files I thought would be needed, it still didnt work (couldn't figure out why).

Git seems overkill so I used the ZIP route.